### PR TITLE
refactor(@ui/settings): add advanced settings section to storybook

### PR DESCRIPTION
**Pull request by iurimatias**
### What does the PR do

Adds the advanced settings section to storybook

### CODE REVIEW

File: storybook/pages/AdvancedSettingsPage.qml [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+
+                function setWakuV2LightClientEnabled(mode) {
+                    logs.logEvent("advancedStore::setWakuV2LightClientEnabled", ["mode"], arguments)
+                    mockData.wakuV2LightClientEnabled = mode
+                }
+                function setBloomLevel(mode) {
+                    logs.logEvent("advancedStore::setBloomLevel", ["mode"], arguments)
+                    mockData.bloomLevel = mode
+                }
+                function toggleDebug() {
```
Comment (noeliaSD) [inline storybook/pages/AdvancedSettingsPage.qml:91]: Something is wrong when trying to switch `debug` option:
"QML ConfirmationDialog: cannot find any window to open popup in."

File: storybook/pages/AdvancedSettingsPage.qml [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+                    logs.logEvent("advancedStore::toggleDebug")
+                    mockData.isDebugEnabled = !mockData.isDebugEnabled
+                }
+                function toggleAutoMessage() {
```
Comment (noeliaSD) [inline storybook/pages/AdvancedSettingsPage.qml:95]: Something is wrong when trying to switch `auto message` option:
"QML ConfirmationDialog: cannot find any window to open popup in."

File: storybook/pages/AdvancedSettingsPage.qml [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+                    logs.logEvent("advancedStore::toggleAutoMessage")
+                    mockData.isAutoMessageEnabled = !mockData.isAutoMessageEnabled
+                }
+                function toggleTelemetry() {
```
Comment (noeliaSD) [inline storybook/pages/AdvancedSettingsPage.qml:99]: Something is wrong when trying to switch `telemetry` option:
"QML ConfirmationDialog: cannot find any window to open popup in."

**Review by noeliaSD — CHANGES_REQUESTED** [review]
Nice that this project is growing!! :)

Added some comments here.

File: storybook/pages/AdvancedSettingsPage.qml (hunk 1 above)
Comment (iurimatias) [reply storybook/pages/AdvancedSettingsPage.qml:91]: yes, it's an issue we have with (some) modals in storybook in general and will have to be fixed in some subsquent PR. It seems in part due to the way our modals work which is not consistent

**Pull request merged**